lib/composables/Card.ts

```diff
@@ -22,12 +22,15 @@ export interface Footer {
 export interface Action {
   type: 'primary' | 'secondary' | 'tertiary' | 'text' | 'mute'
   mode?: Mode
-  icon?: any
+  icon?: ActionIconType | object
+  disabled?: string | null
   label?: string
   link?: string
   callback?(): void
 }
 
+export type ActionIconType = 'add' | 'edit' | 'delete'
```

Comment:
Let's change the option name to match the icon file 🙏 

```suggestion
export type ActionIconType = 'plus' | 'edit-3' | 'trash-2'
```